backend/cli.py

```diff
@@ -35,7 +35,7 @@
 
 
 @app.command()
-def add_imdb_ratings():
+def add_imdb_ratings(min_votes: int = 100):
```

Comment:
I'm not sure if we'll miss a lot with a minimum of 100 - Maybe 25 is enough

Reply:
i did 50 :D